src/stores/RoomViewStore.tsx

```diff
@@ -353,6 +355,22 @@ export class RoomViewStore extends EventEmitter {
                 });
             }
 
+            // Start a call if requested and not already there
+            if (payload.view_call && room && !isVideoRoom(room)) {
+                if (CallStore.instance.getCall(payload.room_id) === null)
+                    ElementCall.create(room, false).then(() => {
+                        const call = CallStore.instance.getCall(payload.room_id!)!;
+                        call.presented = true;
```

Comment:
Do we really only need to call this is if `CallStore.instance.getCall(payload.room_id) === null`
If there is already a rtc session and therefore `getCall() !== null` we still want to set presented to true + call `call.start()`.

Reply:
Right you are - fixed